oshi-core/src/main/java/oshi/software/os/unix/freebsd/FreeBsdOSProcess.java

```diff
@@ -332,25 +333,14 @@ private int queryBitness() {
 
     @Override
     public List<OSThread> getThreadDetails() {
-        List<OSThread> threads = new ArrayList<>();
         String psCommand = "ps -awwxo " + PS_THREAD_COLUMNS + " -H";
         if (getProcessID() >= 0) {
             psCommand += " -p " + getProcessID();
         }
-        List<String> threadList = ExecutingCommand.runNative(psCommand);
-        if (threadList.size() > 1) {
-            // remove header row
-            threadList.remove(0);
-            // Fill list
-            for (String thread : threadList) {
-                Map<PsThreadColumns, String> threadMap = ParseUtil.stringToEnumMap(PsThreadColumns.class, thread.trim(),
-                        ' ');
-                if (threadMap.containsKey(PsThreadColumns.PRI)) {
-                    threads.add(new FreeBsdOSThread(getProcessID(), threadMap));
-                }
-            }
-        }
-        return threads;
+        Predicate<Map<PsThreadColumns, String>> columnPRI = threadMap -> threadMap.containsKey(PsThreadColumns.PRI);
```

Comment:
(Preference) let's name this for the reason it exists, e.g., `hasAllPSColumns`

Reply:
would `hasColumnsPri` suffice? im not sure what `AllPS` means i.e. the other ones would be `hasKeywordsArgs` etc...